example/src/components/StatsModal.tsx

```diff
@@ -0,0 +1,94 @@
+import { AdditionalColors, BrandColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import React, { useCallback, useEffect, useState } from 'react';
+import { ActivityIndicator, View, StyleSheet } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { ReactNativeModal } from 'react-native-modal';
+
+import { Stats } from './Stats';
+import { InCallButton } from './buttons/InCallButton';
+
+type StatsModalProps = {
+  visible: boolean;
+  onClose: () => void;
+};
+
+export const StatsModal = ({ visible, onClose }: StatsModalProps) => {
+  const [statsIntervalID, setStatsIntervalID] = useState<number>(0);
```

Comment:
don't put this in the state, it doesn't have to do anything with re-rendering

you can put it in ref or better yet just cleanup (return function that clears the interval) in the useEffect that sets the interval (if possible)

Reply:
Removed.